### PR TITLE
PRO-1111: Use Ethereum chain ID for Aurora native token rates

### PR DESCRIPTION
**Pull request by Yash-Chaubal**
Now getting rates for Aurora's native token from Ethereum Mainnet

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by poocart — APPROVED** [review]
@Yash-Chaubal lgtm, but please if u could add simple explanation in comment on why this has been chosen, thanks! 🙏

File: src/providers/EtherspotContextProvider.tsx [hunk 1]
```diff
@@ -425,9 +425,11 @@ const EtherspotContextProvider = ({
   const getRatesByNativeChainId = async (chainId: number) => {
     if (!sdk) return null;
     try {
+      const nativeTokenChainId = chainId === CHAIN_ID.AURORA ? CHAIN_ID.ETHEREUM_MAINNET : chainId;
```
Comment (poocart) [inline src/providers/EtherspotContextProvider.tsx:428]: can we get a comment on why this is a case, thx!

**Review by poocart — APPROVED** [review]
lgtm, only one comment to consider

**Comment by Yash-Chaubal** [comment]
Ah yes sorry, so basically currently the backend returns the wrong rates for the aurora native tokens, so I made these changes to always give us the rates of ETH.

The backend now has some fixes in QA, so once they get pushed to prod these changes should be irrelevant, and we can delete this PR

**Comment by Yash-Chaubal** [comment]
Closed as the issue has been solved in the backend, and should no longer need this workaround once the changes have been pushed onto prod.

**Pull request closed without merging**